src/openrct2/scripting/ScriptEngine.cpp

```diff
@@ -1050,15 +1055,36 @@ GameActions::Result ScriptEngine::QueryOrExecuteCustomGameAction(std::string_vie
             return action;
         }
 
+        std::vector<DukValue> pluginCallArgs;
+        if (GetTargetAPIVersion() <= API_VERSION_68_CUSTOM_ACTION_ARGS)
```

Comment:
So even though you said the change is breaking, that is not really the case, since we have a fallback mechanism, correct?

Reply:
It's still a breaking chance for plugin developers who wish to increase their targeted API, so IMHO it should be documented as one.